tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,43 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+from base import Base
+
+
+class Collections(Base):
+    """ Collections page. """
+
+    _wait_element = (By.CLASS_NAME, 'collections')
+
+    def wait_for_page_to_load(self):
+        self.wait.until(EC.presence_of_all_elements_located(
+            (self._wait_element)))
+        return self
+
+    @property
+    def featured(self):
+        return self.Featured(self)
+
+    class Featured(Region):
+        """
+        Represents the featured collection region.
+        """
+        _root_locator = (By.CLASS_NAME, 'island')
+
+        _collection_name = (By.CSS_SELECTOR, "h2.collection > span")
+
+        @property
+        def collections(self):
+            return [Collections.Collection(self.page, el)
+                    for el in self.find_elements(*self._collection_name)]
+
+    class Collection(Region):
+        """
+        Represents an individual collection.
+        """
+        _root_locator = (By.CLASS_NAME, 'items')
+
+        @property
+        def title(self):
```

Comment:
Nit: I'd call this name instead of title. Using title in page objects could cause confusion with the document's title.